conf/core/ConfigOptions.asciidoc

```diff
@@ -4353,9 +4398,24 @@ SampledAngleHistogramExtractor
 
 The way joiner implementation used by the `WayJoinerOp` to repair way splits after a conflation job.
 
-This operation can take on the following two options:
-* `hoot::WayJoinerBasic`
+This operation can take on the following options:
+* `hoot::ReplacementSnappedWayJoiner`
 * `hoot::WayJoinerAdvanced`
+* `hoot::WayJoinerBasic`
+
```

Comment:
`ReplacementSnappedWayJoiner` is the way joining I came up with to work within the changeset replacement workflow. It really probably wouldn't be used anywhere else.